styles.css

```diff
@@ -1291,7 +1291,7 @@ h1 {
 
 .btn:active {
   opacity: 1;
-  color: #5E6C84;
+  color: #5e6c84;
 }
```

Comment:
Please consider adding the functionality of `hover and press` on the button. So when the button is hovered or pressed its style changes as follows. To achieve this you can add the following functionality of code into your CSS file. Currently, your styles don't match the Figma design.

## Solution:--

```
.btn:hover{
    background-color: #6070FF;
    color: white;
}
```

```
.btn:active{
    background-color: #2230D2;
    color: white;
}
```

## Demonstration:--

![image](https://user-images.githubusercontent.com/94223367/219416172-4958d9ca-1fa4-4068-adbb-ef3657bc7b3f.png)